dre_probe_matcher.py

```diff
@@ -6,8 +6,9 @@
 from decouple import config 
 
 SEND_TO_MONGO = True # send all raw and calculated data to the mongo db if true
-RUN_ALIGNMENT = True # send data to servers to calculate alignment if true
+RUN_ALIGNMENT = False # send data to servers to calculate alignment if true
```

Comment:
Is this a deliberate change to the default?  Or just something you wanted to do locally?